lib/utilities/healthcheckHandler.js

```diff
@@ -0,0 +1,92 @@
+import { errors, ipCheck } from 'arsenal';
+import _config from '../Config';
+import data from '../data/wrapper';
+import vault from '../auth/vault';
+import metadata from '../metadata/wrapper';
+import async from 'async';
+
+// current function utility is minimal, but will be expanded
+export function isHealthy() {
+    return true;
+}
+
+function writeResponse(res, error, log, results, cb) {
+    let statusCode = 200;
+    if (error) {
+        // error.code can be a string (such as ECONNREFUSED)
+        if (Number.isInteger(error.code)) {
+            statusCode = error.code;
+        } else {
+            statusCode = 500;
+        }
+    }
+    // If we're here then s3 is considered ok
+    const initial = { S3: { code: 200, message: 'OK' } };
```

Comment:
If you can move this logic to clientCheck, this method can become generic enough that it can be reused. For example, my upcoming PR is adding additional stats to /_/healthcheck route. I could re-use this method.